src/Database/Adapter/MariaDB.php

```diff
@@ -1311,7 +1312,7 @@ public function getAttributeWidth(Document $collection): int
                     $total += 19; // 2022-06-26 14:46:24
                     break;
                 default:
-                    throw new Exception('Unknown Type');
+                    throw new DatabaseException('Unknown Type. Must be one of ${Database::VAR_INTEGER}, ${Database::VAR_FLOAT}, ${Database::VAR_BOOLEAN}, ${Database::VAR_DOCUMENT}, ${Database::VAR_DATETIME}');
```

Comment:
```suggestion
                    throw new DatabaseException("Unknown Type. Must be one of ${Database::VAR_INTEGER}, ${Database::VAR_FLOAT}, ${Database::VAR_BOOLEAN}, ${Database::VAR_DOCUMENT}, ${Database::VAR_DATETIME}");
```

String interpolation only works when the string is enclosed in double quotes